tests/openwisp2/sample_config/migrations/0003_update_vpn_dhparam_length.py

```diff
@@ -0,0 +1,16 @@
+from django.db import migrations
+from django.contrib.auth.models import Permission
+
+from openwisp_controller.config.migrations import update_vpn_dhparam_length
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('sample_config', '0002_default_groups_permissions'),
+    ]
+
+    operations = [
+        migrations.RunPython(
+            update_vpn_dhparam_length, reverse_code=migrations.RunPython.noop
+        ),
+    ]
```

Comment:
@nemesisdesign This migration is in `sample_app`, in a previous PR, you mentioned to add a similar small change in existing migrations in sample app to avoid creating another file to keep it simple; however, since this is not related to the previous migration, I have added a new migration.
Let me know if in this case you would prefer to change the name of previous migration and add it in that migration itself.

Reply:
I think we can simply do without the data migration, the sample app is not meant to be used in production system so we don't need to upgrade its data